aesara/tensor/math_opt.py

```diff
@@ -3079,6 +3019,8 @@ def _is_1(expr):
     allow_multiple_clients=True,
     values_eq_approx=values_eq_approx_remove_inf,
     skip_identities_fn=_skip_mul_1,
+    tracks=[sigmoid],
```

Comment:
For the `PatternSub`, the new tracks is no more restrictive than before as the internal matching logic is all based on the restrictive Op equality

Reply:
Then it would seem like this change isn't necessary and only adds more code that we would need to change in the future, no?